### ISSUE TITLE
Nexus 6 crash on UK - Driving Licence

### ISSUE THREAD

**Issue by damienlo** [BlinkID/blinkid-android#147]
Hello,

### Description

Each time I am trying to scan a UK Driving licence it crashes.
Stack trace:

```
2019-03-11 10:13:45.517 21723-22481/com.yoti.mobile.android.staging.debug A/libc: Fatal signal 11 (SIGSEGV), code 1, fault addr 0xfffffffc in tid 22481 (Recognition)
2019-03-11 10:13:45.577 22544-22544/? A/DEBUG: *** *** *** *** *** *** *** *** *** *** *** *** *** *** *** ***
2019-03-11 10:13:45.577 22544-22544/? A/DEBUG: Build fingerprint: 'google/shamu/shamu:7.0/NBD92G/3754298:user/release-keys'
2019-03-11 10:13:45.577 22544-22544/? A/DEBUG: Revision: '0'
2019-03-11 10:13:45.577 22544-22544/? A/DEBUG: ABI: 'arm'
2019-03-11 10:13:45.577 22544-22544/? A/DEBUG: pid: 21723, tid: 22481, name: Recognition  >>> com.yoti.mobile.android.staging.debug <<<
2019-03-11 10:13:45.577 22544-22544/? A/DEBUG: signal 11 (SIGSEGV), code 1 (SEGV_MAPERR), fault addr 0xfffffffc
2019-03-11 10:13:45.577 22544-22544/? A/DEBUG:     r0 00000000  r1 00000000  r2 a7d7c2fc  r3 00000000
2019-03-11 10:13:45.578 22544-22544/? A/DEBUG:     r4 7c93f894  r5 7c93f840  r6 00000001  r7 7c07a494
2019-03-11 10:13:45.579 22544-22544/? A/DEBUG:     r8 7c93f7d0  r9 7c9d6ec0  sl 00000001  fp 795b2080
2019-03-11 10:13:45.579 22544-22544/? A/DEBUG:     ip a7d72948  sp 7c079c48  lr 7c6eb999  pc 7c78f790  cpsr 600f0010
2019-03-11 10:13:45.580 22544-22544/? A/DEBUG: backtrace:
2019-03-11 10:13:45.580 22544-22544/? A/DEBUG:     #00 pc 00295790  /data/app/com.yoti.mobile.android.staging.debug-1/lib/arm/libBlinkID.so
```

### Environment Details

**BlinkID version:** 4.7

**Device model:** Nexus 6

**Device Android version:** 7.0

**Device ABI (processor architecture, e.g. ARMv7):** ARM

**Comment by i1E** [comment]
Hi @damienlo,

Thank you for the issue report,

we have reproduced the crash on the Nexus 6 and we will fix it in the next SDK version (4.8.0) which is planned for the end of March.

**Comment by damienlo** [comment]
awesome! Cheers
